grabbit/src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitCleanJobRepositoryServlet.groovy

```diff
@@ -0,0 +1,27 @@
+package com.twcable.grabbit.client.servlets
```

Comment:
Yeah temporary. I was thinking about how to incorporate triggering this actual functionality. I know we talked about having a Cron scheduler before but I am wondering if Grabbit should have a DELETE API .. which then can be triggered externally via a Cron (may be it can be controlled better that way?) cc @jdigger